src/authentication.ts

```diff
@@ -169,3 +180,37 @@ export class OAuthAuthentication implements Authentication {
         });
     }
 }
+
+/**
+ * Certificate service Authentication class. Provides the certificate and the private key for the building of the keystore.
+ */
+export class CertificateServiceAuthentication {
```

Comment:
Rename to CertificateAuthentication and fix all documentation (including comments) related to it